test/SimpleStore.ts

```diff
@@ -85,12 +116,12 @@ export class SimpleStore extends StoreBase {
     // Note: explicitly adding decorator so the tests always works, even outside of debug mode. This is not necessary
     // in real stores, as explained above clearStoreData.
     @warnIfAutoSubscribeEnabled
-    setStoreData(id: string, storeData: StoreData) {
+    setStoreData(id: string, triggerKey: string, storeData: StoreData) {
```

Comment:
Suggestion: Make `triggerKey = id` since many usages just want `id`. If you put it at the end then they don't need to specify it, or you can put it in the middle and use `undefined` to get `id`.
```ts
setStoreData(id: string, storeData: StoreData, triggerKey = id) {...}

setStoreData('id', data);
```
```ts
setStoreData(id: string, triggerKey = id, storeData: StoreData) {...}

setStoreData('id', undefined, data);
```

Reply:
Nah I want people doing the tests to have to think about it